ihp.nix

```diff
@@ -134,6 +136,8 @@ mkDerivation {
     unagi-chan
     with-utf8
     ihp-hsx
+    xml-conduit
```

Comment:
Not a fan of bringing in these heavy dependencies into IHP core. Is there another solution? Otherwise maybe we can make this a plugin?

Reply:
Ah, yes! I remembered that Blaze supports custom tags and attributes, no new deps and faster.